### DISCUSSION TITLE
Auto seat heat commands

### DISCUSSION THREAD

**Discussion by SylvainGa** [timdorr/tesla-api#544] [Q&A] [answered]
I noticed that climate_state now returns auto_seat_climate_left and auto_seat_climate_right (true/false). Did anyone find out what command in to use to turn Auto on/off? From what I read, it should be available soon if not already.

Thanks.

**Comment by tdorssers** [comment, marked as answer]
The endpoint REMOTE_AUTO_SEAT_CLIMATE_REQUEST is used for that with parameters| `auto_seat_position` and `auto_climate_on` with values 1 or 2 and `true` or `false`.

**Reply by SylvainGa** [reply]
Thanks!

**Comment by SylvainGa** [comment]
What firmware version is this supported for? I'm at 2022.4.5.4 and a call to url https://owner-api.teslamotors.com/api/1/vehicles/{removed}/command/remote_auto_seat_climate_request with options sets to {auto_climate_on=>true, auto_seat_position=>1}

returns a 200 but the seat do not switch to Automatic, and yes, the climate was turned on. I can set the heat level and that's it :-(

**Reply by tdorssers** [reply]
I'm at 2022.8.2 and for me it's working.

**Reply by SylvainGa** [reply]
Ok, I'll retry once I finally get that update.

**Reply by SylvainGa** [reply]
I'm at 2022.4.5.17 and it is now working :-)
